authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc

```diff
@@ -0,0 +1,615 @@
+[id="cco-mode-sts"]
+= Using STS mode
+include::modules/common-attributes.adoc[]
+:context: cco-mode-sts
+toc::[]
+
+STS mode is supported for Amazon Web Services (AWS).
+
+:FeatureName: Support for AWS Secure Token Service (STS)
+include::modules/technology-preview.adoc[leveloffset=+0]
+
+In STS mode, the Cloud Credential Operator (CCO) uses AWS Secure Token Service (STS) to assign components IAM roles that provide short-term, limited-privilege security credentials.
+
+Requests for new and refreshed credentials are automated by using an AWS IAM OpenID Connect (OIDC) identity provider. {product-title} signs service account tokens that are trusted by AWS IAM, and can be projected into a pod and used for authentication. Tokens are refreshed after one hour.
+
+.STS authentication flow
+image::sts_flow.png[STS authentication flow]
+
+The implementation of STS mode changes the format of the AWS credentials root secret in the `kube-system` namespace.
+
+.AWS secret format using a mode other than STS
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: kube-system
+  name: aws-creds
+data:
+  aws_access_key_id: <base64-encoded-access-key-id>
+  aws_secret_access_key: <base64-encoded-secret-access-key>
+----
+
+.AWS secret format using STS mode
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: kube-system
+  name: aws-creds
+data:
+  role_name: <operator-role-name> <1>
+  web_identity_token_file: <path-to-token> <2>
+----
+<1> The IAM role for the cluster
++
+<needinfo> Is this accurate/informative? or is there a better way to describe it?
+<2> The path to the service account tokens.
+
+[id="sts-mode-installing"]
+== Installing an {product-title} cluster configured for STS mode
+
+In {product-title} version 4.7, installing a cluster that is configured to use the CCO in STS mode requires the following steps:
+
+. xref:#sts-mode-installing-preparing[Preparing the installer]
+. Creating AWS resources xref:#sts-mode-installing-config-preflight[using the `sts-preflight` tool] or xref:#sts-mode-installing-config-manual[manually]
+. xref:#sts-mode-installing-running-installer[Running the installer]
+. xref:#sts-mode-installing-verifying[Verifying the installation]
+
+[NOTE]
+====
+<needinfo>
+Are they basically following the link:https://docs.openshift.com/container-platform/4.6/installing/installing_aws/installing-aws-customizations.html#installation-initializing_installing-aws-customizations[Installing a cluster on AWS with customizations] procedure here, with some specific differences for STS (and of course the addition of the preflight stuff)?
+====
+
+
+[id="sts-mode-installing-preparing"]
+=== Preparing the installer
+
+.Prerequisites
+* Obtain the pull secret for your cluster.
+
+.Procedure
+
+. Set the `$RELEASE_IMAGE` environment variable to use a sufficiently new {product-title} release:
++
+[source,terminal]
+----
+<needinfo>
```

Comment:
"Sufficiently new" was back when there were still missing pieces that hadn't merged into 4.7. Now that 4.7 has all the needed PRs, you only need OpenShift version 4.7.